src/woo_publications/metadata/models.py

```diff
@@ -58,6 +61,43 @@ class InformationCategory(OrderedModel):
         max_length=15,
         default=InformationCategoryOrigins.custom_entry,
     )
+    # Retention fields:
+    bron_bewaartermijn = models.CharField(
+        _("retention source"),
+        help_text=_(
+            "The source of the retention (example: Selectielijst gemeenten 2020)."
+        ),
+        max_length=255,
+        null=True,
+        blank=False,
+    )
+    selectiecategorie = models.CharField(
+        _("selection category"),
+        help_text=_(
+            "The category within the source of the retention (example: 20.1.2)."
+        ),
+        max_length=255,
+        blank=True,
+    )
+    archiefnominatie = models.CharField(
+        _("archive nomination"),
+        help_text=_("Determines if the archived data will be retained or destroyed."),
+        choices=ArchiveNominationChoices.choices,
+        max_length=16,
+        null=True,
+        blank=False,
+    )
```

Comment:
```suggestion
    archiefnominatie = models.CharField(
        _("archive action"),
        help_text=_("Determines if the archived data will be retained or destroyed."),
        choices=ArchiveNominationChoices.choices,
        max_length=16,
        blank=False,
        default=ArchiveNominationChoices.retain,
    )
```
* "archive action" is a better description of what needs to happen
* here too, don't use `null=True` on `models.CharField`
* the required field must have a default for existing data -> I've gone for the non-destructive option.

As an aside, picking a `max_length` equal to the longest choice value is not very future proof if the choices get extended at some point. Often I just go with `50` chars to have some headroom, the DB space it takes is negligible anyway.